Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -134,6 +134,6 @@
   "loc.messages.NGCommon_UnableToFindTool": "Unable to find tool %s",
   "loc.messages.Warning_SessionCreationFailed": "Could not create provenance session: %s",
   "loc.messages.Warning_UpdatingNuGetVersion": "Updating version of NuGet.exe to %s from %s. Behavior changes or breaking changes might occur as NuGet updates to a new version. If this is not desired, deselect the 'Check for Latest Version' option in the task.",
-  "loc.messages.NetCore3Update": "Info: Azure Pipelines hosted agents have been updated to contain .Net Core 3.x (3.1) SDK/Runtime along with 2.1. Unless you have locked down a SDK version for your project(s), 3.x SDK might be picked up which might have breaking behavior as compared to previous versions. \nSome commonly encountered changes are: \nIf you're using `Publish` command with -o or --Output argument, you will see that the output folder is now being created at root directory rather than Project File's directory. To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting",
+  "loc.messages.NetCore5Update": "Info: Azure Pipelines hosted agents have been updated to contain .Net Core 5.x SDK/Runtime along with the older ones currently lts. Unless you have locked down a SDK version for your project(s), 5.x SDK might be picked up which might have breaking behavior as compared to previous versions. \nYou can learn more about the breaking changes here: \n https://docs.microsoft.com/en-us/dotnet/core/tools/ \n https://docs.microsoft.com/en-us/dotnet/core/compatibility/",
```

Comment:
It would still be useful to keep the troubleshooting link

```
To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
```